plasma_framework/contracts/mocks/exits/payment/routers/PaymentInFlightExitRouterMock.sol

```diff
@@ -52,4 +52,7 @@ contract PaymentInFlightExitRouterMock is PaymentInFlightExitRouter {
     function getInFlightExitOutput(uint192 exitId, uint16 outputIndex) public view returns (PaymentExitDataModel.WithdrawData memory) {
         return inFlightExitMap.exits[exitId].outputs[outputIndex];
     }
+
+    function fundForBondPayOut() public payable {
```

Comment:
Minor: there is a same similar function in SE mock: `depositFundForTest`, might be nice to keep name consistent

Reply:
done